pom.xml

```diff
@@ -47,17 +47,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.mortbay.jetty.alpn</groupId>
-            <artifactId>alpn-boot</artifactId>
-            <version>${jetty.alpn.version}</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>org.eclipse.jetty.alpn</groupId>
             <artifactId>alpn-api</artifactId>
             <version>1.1.2.v20150522</version>
-            <!-- Provided by alpn-boot; see
-            http://www.eclipse.org/jetty/documentation/9.2.8.v20150217/alpn-chapter.html#alpn-understanding -->
             <scope>provided</scope>
```

Comment:
I just checked netty (they use the agent, instead of all the profiles too). Looks like the `alpn-api` is still there, as provided:
https://github.com/netty/netty/blob/c295f2239049a677527fa2be7ab14d7e413da4b9/pom.xml#L291

I _guess_ best is, to follow that...
I